pkg/driver/mount.go

```diff
@@ -195,6 +195,11 @@ func passthroughEnv() []string {
 	}
 	webIdentityFile := os.Getenv(webIdentityTokenEnv)
 	awsRoleArn := os.Getenv(roleArnEnv)
+	hostTokenPath := os.Getenv(hostTokenPathEnv)
+	if hostTokenPath == "" {
+		// set the default in case the env variable isn't found
+		hostTokenPath = "/var/lib/kubelet/plugins/s3.csi.aws.com/token"
+	}
 	if webIdentityFile != "" {
```

Comment:
should this be  `==`  in place of `!=` ? if webIdentityFile is set should we not use it?

Reply:
hmm not quite, we're trying to set the env variable here based on the driver env (if you look at `webIdentityFileEnv` it comes from driver.go, this isn't something set by the customer) so if the driver is running with the web token, that should be passed here as well (but with different paths)

Reply:
I see, so this line.
https://github.com/awslabs/mountpoint-s3-csi-driver/blob/main/pkg/driver/driver.go#L78C4-L82
I was little confused as it was pointing to '/csi/token' as path in driver. I assumed this might be some mechanism to override it. Thanks for clarification.